blurb/blurb.py

```diff
@@ -749,6 +749,10 @@ def find_editor():
         editor = os.environ.get(var)
         if editor is not None:
             return editor
+    editor = subprocess.check_output(
+        ["git", "config", "--get", "core.editor"]).strip()
+    if editor:
+        return editor
```

Comment:
> If it has a default value of "vi" anywhere

It's configure option, and default value of the configure option is `vi`.
https://github.com/git/git/blob/8b2efe2a0fd93b8721879f796d848a9ce785647f/configure.ac#L406-L411

At least, Ubuntu uses `editor` as default editor.

But most important fact is it's always same to editor which `git commit` starts.
I believe most blurb users are git user too and they configure git before using blurb.